pkg/types/types.proto

```diff
@@ -30,6 +30,12 @@ message Interface {
 	// Here, bridgeAddr is the address at which the bridge is attached on the root bus,
 	// while deviceAddr is the address at which the network device is attached on the bridge.
 	string pciAddr = 6;
+
+	// Type defines the type of interface described by this structure.
+	// The expected values are the one that are defined by the netlink
+	// library, regarding each type of link. Here is a non exhaustive
+	// list: "veth", "macvtap", "vlan", "macvlan", "tap", ...
+	string type = 7;
```

Comment:
Also please do not call it `type` since it's a reserved word in golang. `link_type` might be better here. Sorry for the late response as I was OOO last week.